redis/tests/test_basic.rs

```diff
@@ -1448,3 +1449,32 @@ fn test_blocking_sorted_set_api() {
         );
     }
 }
+
+#[test]
+fn test_set_client_name_by_config() {
+    const CLIENT_NAME: &str = "TEST_CLIENT_NAME";
+
+    let ctx = TestContext::with_clientname(CLIENT_NAME);
+    let mut con = ctx.connection();
+
+    let client_list: redis::InfoDict = redis::cmd("CLIENT").arg("LIST").query(&mut con).unwrap();
+    assert!(client_list.len() == 1, "Expecting single client, got: {}", client_list.len());
+
+    let val: &redis::Value = client_list.values().next().unwrap();
```

Comment:
can this be parsed to a HashMap, and let the parsing operation find the keys and values?

Reply:
have no idea what you mean, show me if its better

Reply:
not relevant, will try to look into infoDict